service/docs/source/install.rst

```diff
@@ -56,3 +68,32 @@ least as recent as the release candidate branch.  For this reason,
 updates will always come from the development branch download
 repository if both the development repository and the release
 candidate repository are added to your package configuration,
+
+
+Un-packaged Features
+--------------------
+
+The installing the packages described above will not provide the
+:doc:`GEOPM HPC Runtime <runtime>` features (e.g.
+:doc:`geopmlaunch(1) <geopmlaunch.1>`, :doc:`geopm_prof(3) <geopm_prof.3>`,
+:doc:`geopm_report(7) <geopm_report.7>`, ``libgeopm.so``, etc.).
+The GEOPM HPC Runtime must be built against a particular implementation of the
+Message Passing Interface (MPI).  Typically the implementation of MPI is
+specific to the system where the HPC application is being run.  An exception
+to this rule is the OpenHPC distribution.  GEOPM version 1 has been packaged
+with OpenHPC, and we hope to distribute version 2 with OpenHPC in the future.
```

Comment:
Their download page has two release tracks. I think we're in both. If not, maybe specify which.

```suggestion
with `OpenHPC releases <http://openhpc.community/downloads/>`_, and we hope to distribute version 2 with OpenHPC in the future.
```